packages/hub/routes/email-card-drop-requests.ts

```diff
@@ -85,12 +96,46 @@ export default class EmailCardDropRequestsRoute {
 
     ctx.type = 'application/vnd.api+json';
 
+    let prepaidCardMarketV2 = await this.cardpay.getSDK('PrepaidCardMarketV2', this.web3.getInstance());
+
+    if (await prepaidCardMarketV2.isPaused()) {
+      return respondWith503(ctx, 'The prepaid card market contract is paused');
+    }
+
+    let quantityAvailable = await prepaidCardMarketV2.getQuantity(cardDropSku);
+
+    let quantityIsBelowNotificationThreshold = quantityAvailable < notifyWhenQuantityBelow;
```

Comment:
If we only look at `quantityAvailable` without counting active reservations, we could get into a situation where an alert is not triggered despite running out of prepaid cards. 

The situation where we don't get notified is sudden rushes in requesting card drops (We get `>= notifyWhenQuantityBelow` requests, before any of them start claiming).

I think this is not a big deal right now, but probably good to note?

Reply:
I changed the threshold calculation to consider reservations, thanks.